extensions/google-drive/src/utils.ts

```diff
@@ -131,42 +131,13 @@ export const filePreview = async (file: FileInfo | null, controller: AbortContro
 
   const previewPath = await filePreviewPath(file, controller);
   const previewExists = previewPath && existsSync(decodeURI(previewPath).replace("file://", ""));
-  const previewImage = previewExists ? `<img src="${previewPath}" alt="${file.name}" height="200" />` : "";
+  const previewImage = previewExists
+    ? `<img src="${previewPath}" alt="${file.name}" />`
+    : `<img alt="${file.name}" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHg9IjBweCIgeT0iMHB4Igp3aWR0aD0iMTIwIiBoZWlnaHQ9IjEyMCIKdmlld0JveD0iMCAwIDgwIDgwIgpzdHlsZT0iIGZpbGw6IzAwMDAwMDsiPjxwYXRoIGZpbGw9IiNmZmYiIGQ9Ik0xMy41IDc0LjVMMTMuNSAzLjUgNTAuNzkzIDMuNSA2OC41IDIxLjIwNyA2OC41IDc0LjV6Ij48L3BhdGg+PHBhdGggZmlsbD0iIzc4OGI5YyIgZD0iTTUwLjU4Niw0TDY4LDIxLjQxNFY3NEgxNFY0SDUwLjU4NiBNNTEsM0gxM3Y3Mmg1NlYyMUw1MSwzTDUxLDN6Ij48L3BhdGg+PGc+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTUwLjUgMjEuNUw1MC41IDMuNSA1MC43OTMgMy41IDY4LjUgMjEuMjA3IDY4LjUgMjEuNXoiPjwvcGF0aD48cGF0aCBmaWxsPSIjNzg4YjljIiBkPSJNNTEsNC40MTRMNjcuNTg2LDIxSDUxVjQuNDE0IE01MSwzaC0xdjE5aDE5di0xTDUxLDNMNTEsM3oiPjwvcGF0aD48L2c+PC9zdmc+"/>`;
```

Comment:
So that it shows a default file icon when there's no preview available.

It looks like this -

<img src="https://img.icons8.com/officel/120/000000/file.png"/>

Reply:
Shouldn't folders have another image?

![CleanShot 2022-05-19 at 06 58 12@2x](https://user-images.githubusercontent.com/38033107/169211285-03fd96c6-3197-40f4-8f2d-25e5c01d5365.png)

Reply:
Hmm